### PR TITLE
Add read the docs!

### PR DESCRIPTION
**Pull request by lwasser**
Just cause for the love of all docs.

### CODE REVIEW

**Comment by lwasser** [comment]
@mbjoseph or @betatim do you happen to know the pieces that i need to get travis to test build the docs? i looked back to a pr here
https://github.com/earthlab/earthpy/commit/590fa16d9a0117a8a1e0cd65c8e5507376ac07f9#diff-354f30a63fb0907d4ad57269548329e3

to try to begin to understand. travis is failing on dev-requirements.txt but i only saw a makefile in this pr. if i understood the pieces needed to make travis happy it would help me fix this pr!

**Comment by lwasser** [comment]
looks like it is building now but with one error - it's still looking for an old path somewhere. [see here.](https://travis-ci.com/earthlab/abcnb/jobs/166331087#L556)

File: docs/Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
```
Comment (betatim) [inline docs/Makefile]:
```suggestion
SPHINXOPTS    = -W
```

This turns all wanrings into errors. Seems pedantic but in my experience without this you end up with a boatload of warnings over time that train you to look away when they scroll past on the screen.

**Comment by betatim** [comment]
Never had this problem of broken travis links :-/

I went and did what https://github.com/travis-ci/travis-ci/issues/3264#issuecomment-112103362 suggests. Maybe that helps?

**Comment by lwasser** [comment]
thank you @betatim that sync button did seem to work. travis finds the correct name now. thank you! the build is still failing but i just restarted it as the static url atleast has the correct path now.

**Comment by lwasser** [comment]
i have a sneaky suspicion the  `_static` and `_build` dirs were missing because they were empty and git doesn't like empty dirs. so i followed what you did in earthpy @betatim and hope that will fix things. thank you so much!

**Comment by lwasser** [comment]
fixed!! clean build :)

**Pull request merged**